contracts/Hub.sol

```diff
@@ -0,0 +1,138 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.7;
+
+import "@openzeppelin/contracts/access/Ownable.sol";
+
+/**
+ * @title Hub Contract that coordinates Storage, Fund and Minter contract.
+ * @dev The contract can mint new tokens, check the collateral ratio of addresses
+ * and liquidate addresses if the collateral ratio falls below 150%
+ * @author The Everest team.
+ */
+
+contract Hub is Ownable {
+    // Contract addresses
+    Minter private minterContract;
+    Fund private fundContract;
+    Storage private storageContract;
+
+    address[] private authorizedAddresses;
+    mapping(string => address) private tickersymbolToAddress;
```

Comment:
I suggest to use a different variable name because the subname address is to generic `tickersymbolToSynthContract` would declare the purpose of the mapping better